tasks/update-mlnx-ofed.yml

```
@@ -0,0 +1,19 @@
+---
+ - name: Add script content
+   ansible.builtin.copy:
+     dest: /etc/kernel/header_postinst.d/mlnx-ofed-kernel
+     mode: 770
```

`750` should be fine.
```suggestion
     mode: 750
```